flake.lock

```diff
@@ -360,11 +347,11 @@
         "tullia": "tullia"
       },
       "locked": {
-        "lastModified": 1670221914,
-        "narHash": "sha256-7Hu+s2wHiSWjnA1RlskxfV0JLp0Z1D6jGnh3KxmJLOY=",
+        "lastModified": 1677027044,
+        "narHash": "sha256-o21p0i5yXzwC8l6Ddw6qw3yWTTZEIII6HUUohYvMaEE=",
         "owner": "input-output-hk",
         "repo": "haskell.nix",
-        "rev": "6a971848cb717d8d0c8db71bd1c1d9fd38f02851",
+        "rev": "4883bba92116b54db92182ca24841c0aa922c9d6",
```

Comment:
@andreabedini Is this seriously the diff of haskell.nix between these two commits? https://github.com/input-output-hk/haskell.nix/compare/6a971848cb717d8d0c8db71bd1c1d9fd38f02851..4883bba92116b54db92182ca24841c0aa922c9d6 :o

Reply:
Yes it is. It's ~100 of business logic change but it's right at the core of it. This makes a change in the way the cabal plan is represent in nix and, as a consequence, all materialisations have to be redone and all the compiliers and tools end up being rebuilt.